module/actor.js

```diff
@@ -111,29 +111,26 @@ class DCCActor extends Actor {
     }
 
     /** Handle Fumbles * */
-    let fumble = ''
+    const fumble = ''
     let fumbleDie
     try {
       fumbleDie = this.data.data.items.armor.a0.fumbleDie
     } catch (e) {
       fumbleDie = '1d4'
     }
-    if (Number(roll.dice[0].results[0]) === 1) {
```

Comment:
I think this got lost in my rebase. Will fix it up!

Reply:
Heh, I just had to change this to work with Foundry 0.7.0. :(